packages/frontend/editor-ui/src/features/ai/chatHub/ChatView.vue

```diff
@@ -482,7 +483,9 @@ function handleRegenerateMessage(message: ChatHubMessageDto) {
 
 	const messageToRetry = message.id;
 
-	chatStore.regenerateMessage(
+	editingMessageId.value = undefined;
```

Comment:
Unrelated bug, but editing a message should be canceled when regeneration begins, similar to when submitting a new message